apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -213,10 +237,8 @@ export class SendMessageInApp extends SendMessageBase {
         {
           $set: {
             seen: false,
-            cta: step.template.cta,
-            content,
-            payload: messagePayload,
             createdAt: new Date(),
+            ...channelData,
```

Comment:
Ensuring that the full constructed channel data is passed to the Message update logic too.